### PR TITLE
PBM-865: mark incompatible backups in pbm status

### PR DESCRIPTION
**Pull request by defbin**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Logical | Physical |
| ------------- | ------------- | ------------- |
| Sharded psmdb 4.4 | failed | failed |
| Non-sharded psmdb 4.2 | failed | failed |
| Non-sharded psmdb 4.4 | failed | failed |
| New cluster psmdb 5.0 | failed | - |
| Non-sharded psmdb 5.0 | failed | failed |
| New cluster psmdb 4.2 | failed | - |
| New cluster psmdb 4.4 | failed | - |
| Sharded psmdb 4.2 | failed | failed |
| Sharded psmdb 5.0 | failed | failed |
| Single-node psmdb 4.2 | failed | failed |
| Single-node psmdb 4.4 | failed | failed |
| Single-node psmdb 5.0 | failed | failed |

**Comment by JNKPercona** [comment]
| Test name  | Logical | Physical |
| ------------- | ------------- | ------------- |
| New cluster psmdb 4.2 | failed | - |
| New cluster psmdb 5.0 | failed | - |
| Non-sharded psmdb 4.4 | failed | failed |
| Non-sharded psmdb 4.2 | failed | failed |
| New cluster psmdb 4.4 | failed | - |
| Non-sharded psmdb 5.0 | failed | failed |
| Single-node psmdb 4.2 | failed | failed |
| Sharded psmdb 4.2 | failed | failed |
| Single-node psmdb 4.4 | failed | failed |
| Sharded psmdb 4.4 | failed | failed |
| Sharded psmdb 5.0 | failed | failed |
| Single-node psmdb 5.0 | failed | failed |

**Comment by JNKPercona** [comment]
| Test name  | Logical | Physical |
| ------------- | ------------- | ------------- |
| Single-node psmdb 5.0 | failed | failed |
| New cluster psmdb 4.4 | failed | - |
| New cluster psmdb 5.0 | failed | - |
| Non-sharded psmdb 4.2 | failed | failed |
| Single-node psmdb 4.2 | failed | failed |
| Single-node psmdb 4.4 | failed | failed |
| Non-sharded psmdb 4.4 | failed | failed |
| New cluster psmdb 4.2 | failed | - |
| Non-sharded psmdb 5.0 | failed | failed |
| Sharded psmdb 4.2 | failed | failed |
| Sharded psmdb 5.0 | failed | failed |
| Sharded psmdb 4.4 | failed | failed |

**Comment by JNKPercona** [comment]
| Test name  | Logical | Physical |
| ------------- | ------------- | ------------- |
| New cluster psmdb 4.4 | failed | - |
| New cluster psmdb 4.2 | failed | - |
| New cluster psmdb 5.0 | failed | - |
| Non-sharded psmdb 4.2 | failed | failed |
| Non-sharded psmdb 4.4 | failed | failed |
| Non-sharded psmdb 5.0 | failed | failed |
| Single-node psmdb 4.2 | failed | failed |
| Single-node psmdb 4.4 | failed | failed |
| Single-node psmdb 5.0 | failed | failed |
| Sharded psmdb 4.2 | failed | failed |
| Sharded psmdb 4.4 | failed | failed |
| Sharded psmdb 5.0 | failed | failed |

File: cli/backup_test.go [hunk 1]
```diff
@@ -177,6 +176,193 @@ func TestBcpMatchCluster(t *testing.T) {
...
+		// 			{Name: "rs1"},
+		// 		},
+		// 	},
+		// 	rsMap: map[string]string{
+		// 		"rs1": "rs0",
+		// 	},
+		// 	expected: errMissedReplsets{
+		// 		replsets: []string{"rs1"},
+		// 	},
+		// },
```
Comment (dAdAbird) [inline cli/backup_test.go:250]: do we need this?

**Review by dAdAbird — DISMISSED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Logical | Physical |
| ------------- | ------------- | ------------- |

**Comment by JNKPercona** [comment]
| Test name  | Logical | Physical |
| ------------- | ------------- | ------------- |
| New cluster psmdb 4.2 | passed | - |
| Non-sharded psmdb 4.2 | failed | passed |
| New cluster psmdb 4.4 | passed | - |
| Non-sharded psmdb 4.4 | failed | passed |
| Non-sharded psmdb 5.0 | failed | failed |
| New cluster psmdb 5.0 | passed | - |
| Single-node psmdb 4.2 | failed | passed |
| Single-node psmdb 4.4 | failed | passed |
| Single-node psmdb 5.0 | failed | passed |
| Sharded psmdb 4.2 | failed | passed |
| Sharded psmdb 4.4 | failed | passed |
| Sharded psmdb 5.0 | failed | failed |

**Comment by JNKPercona** [comment]
| Test name  | Logical | Physical |
| ------------- | ------------- | ------------- |
| New cluster psmdb 4.2 | passed | - |
| New cluster psmdb 4.4 | passed | - |
| New cluster psmdb 5.0 | passed | - |
| Non-sharded psmdb 4.4 | passed | passed |
| Non-sharded psmdb 4.2 | failed | passed |
| Non-sharded psmdb 5.0 | failed | passed |
| Single-node psmdb 4.2 | passed | passed |
| Single-node psmdb 5.0 | failed | passed |
| Single-node psmdb 4.4 | passed | passed |
| Sharded psmdb 4.2 | failed | passed |
| Sharded psmdb 4.4 | failed | passed |
| Sharded psmdb 5.0 | failed | passed |

**Comment by JNKPercona** [comment]
| Test name  | Logical | Physical |
| ------------- | ------------- | ------------- |
| New cluster psmdb 4.2 | passed | - |
| New cluster psmdb 5.0 | passed | - |
| Sharded psmdb 4.2 | failed | passed |
| Sharded psmdb 4.4 | failed | passed |
| Sharded psmdb 5.0 | failed | passed |

**Comment by JNKPercona** [comment]
| Test name  | Logical | Physical |
| ------------- | ------------- | ------------- |
| New cluster psmdb 4.2 | passed | - |
| New cluster psmdb 5.0 | passed | - |
| New cluster psmdb 4.4 | passed | - |
| Non-sharded psmdb 4.2 | passed | passed |
| Non-sharded psmdb 4.4 | passed | passed |
| Non-sharded psmdb 5.0 | failed | failed |
| Single-node psmdb 4.2 | passed | passed |
| Single-node psmdb 4.4 | passed | passed |
| Single-node psmdb 5.0 | failed | passed |
| Sharded psmdb 4.2 | failed | failed |
| Sharded psmdb 4.4 | failed | passed |
| Sharded psmdb 5.0 | failed | passed |

**Review by dAdAbird — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Logical | Physical |
| ------------- | ------------- | ------------- |
| New cluster psmdb 4.2 | passed | - |
| New cluster psmdb 4.4 | passed | - |
| New cluster psmdb 5.0 | passed | - |
| Non-sharded psmdb 5.0 | failed | passed |
| Non-sharded psmdb 4.2 | failed | passed |
| Non-sharded psmdb 4.4 | passed | passed |
| Single-node psmdb 4.2 | passed | passed |
| Single-node psmdb 4.4 | passed | passed |
| Single-node psmdb 5.0 | failed | passed |
| Sharded psmdb 4.2 | failed | passed |
| Sharded psmdb 4.4 | failed | passed |
| Sharded psmdb 5.0 | failed | passed |

**Pull request merged**